<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEncrypt</class>
 <widget class="QMainWindow" name="FileEncrypt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileEncrypt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/icon.icns</normaloff>:/Images/icon.icns</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="modeSelector">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>10</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="modeLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>12</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="strengthLabel">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>12</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Strenght</string>
    </property>
   </widget>
   <widget class="QComboBox" name="strengthSelector">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>10</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>256 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128 bit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="InstructionLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="destinationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="destinationPathLine">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPathButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
